bin/all_sky_search/pycbc_add_statmap

```diff
@@ -611,7 +611,7 @@ while True:
             sep_fg_data[key].data['stat'],
             sep_bg_data[key].data['decimation_factor'],
             bg_t_y,
-            **significance_dict[ifo_combo_key])
+            **significance_dict[key])
```

Comment:
This is definitely a bug, I wonder how it managed to work apparently reasonably well before

Reply:
I think if the event is either not super-loud, or a single in single time, the numbers we get out will seem sensible.

This will only show up after the first round of hierarchical removal as well, so we need two very-loud things in a chunk for this to show up!